Marlin/Configuration.h

```diff
@@ -413,6 +413,13 @@
 #define TEMP_SENSOR_5 0
 #define TEMP_SENSOR_BED 0
 #define TEMP_SENSOR_CHAMBER 0
```

Comment:
Couldn't the `TEMP_SENSOR_PROBE` be a good sensor for the chamber if i gets enough airflow?

Reply:
I think no. When first layers are done it's too near to bed

Reply:
No, I have a MK3S print head setup and the hotend-cooler blows directly onto the probe it either cools or heats up the probe significantly (depending on hotend temp). Also on first layers the bed heats up the probe (this is how the calibration process heats it up). I also have an external chamber sensor that shows completely different readings almost all of the time.